solidity/test/UpdatableConsumer_test.js

```diff
@@ -1,14 +1,19 @@
 import {
   assertActionThrows,
+  bigNum,
   defaultAccount,
   deploy,
   getLatestEvent,
   increaseTime5Minutes,
+  linkContract,
+  keccack,
   oracleNode,
-  toWei
+  toUtf8,
+  toWei,
+  transferLINK,
 } from './support/helpers'
```

Comment:
Optional: We're importing many things from helpers, at this point, does it make sense to collapse the import to:
```
import * as h from './support/helpers'
...
h.toWei(...)
```

This might be something for the future rather than this PR.

Reply:
It's just a honking big regexp search-and-replace, and hitting "yes" or "no" a couple of times. :smile:  d4687ded